src/TwigComponent/src/ComponentRenderer.php

```diff
@@ -44,9 +45,14 @@ public function createAndRender(string $name, array $props = []): string
 
     public function render(MountedComponent $mounted): string
```

Comment:
So the stack will always be empty after the "primary render" is finished?

Reply:
Yup, that's how I designed it. No reason, at the moment, to need to know which components have rendered earlier in this request, for example.